catalog/spatial/csw/spatial-csw-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -220,6 +220,101 @@
 
     </OCD>
 
+    <OCD name="CSW Transactional Profile Federated Source" id="Csw_Transactional_Federated_Source"
```

Comment:
Should you add the corresponding docs to distribution/docs/src/main/resources/_contents/_tables/ ? see @ricklarsen for more info on these tables